pkg/terraformer/types.go

```diff
@@ -29,6 +29,8 @@ const (
 	Validate Command = "validate"
 	// Plan is the terraform `plan` command.
 	Plan Command = "plan"
+	// TerraformerFinalizer is the finalizer used by the terraformer on the terraform configmaps and secrets
+	TerraformerFinalizer = "terraformer.gardener.cloud/state"
```

Comment:
It's not only about protecting the state but also the configuration, isn't it? That's why I would keep the name of the finalizer generic.
```suggestion
	TerraformerFinalizer = "terraformer.gardener.cloud"
```

Reply:
I remember why I used `*/state`, it is because of
```
Invalid value: \"terraformer.gardener.cloud\": name is neither a standard finalizer name nor is it fully qualified
```

I will switch to `gardener.cloud/terraformer`.